src/plugins/downloader/templates/download-manager.tsx

```
@@ -0,0 +1,479 @@
+  // Listen for toggle event from title bar
+  onMount(() => {
+    const handleToggle = () => setIsOpen(!isOpen());
+    window.addEventListener('ytmd-download-manager-toggle', handleToggle);
+    onCleanup(() => window.removeEventListener('ytmd-download-manager-toggle', handleToggle));
+  });
+
+  // Show toast notification when new downloads start
+  createEffect(() => {
+    const currentItems = items();
+    const currentQueueLength = currentItems.filter(
+      (i) => i.status === 'queued' || i.status === 'downloading'
```

ESLint

[error] Insert `,`

(prettier/prettier)